File: app/src/main/java/com/examle/curexchange/data/database/DAOs/HistoryDao.java
package com.examle.curexchange.data.database.DAOs;

import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Insert;

import com.examle.curexchange.data.database.entities.HistoryEntity;

@Dao
public interface HistoryDao {

    @Insert
    void insertOneRaw(HistoryEntity historyEntity);
}

File: app/src/main/java/com/examle/curexchange/data/database/dao/HistoryDao.kt
package com.examle.curexchange.data.database.dao

import android.arch.persistence.room.Dao
import android.arch.persistence.room.Insert
import com.examle.curexchange.data.database.entities.HistoryEntity

@Dao
interface HistoryDao {

    @Insert
    fun insertOneRaw(historyEntity: HistoryEntity)
}